autopush/db.py

```diff
@@ -407,6 +421,12 @@ def store_message(self, uaid, channel_id, message_id, ttl, data=None,
                       headers=None, timestamp=None):
         """Stores a message in the message table for the given uaid/channel
         with the message id"""
+        if not uaid:
```

Comment:
r+

I'd love to chain all these `if not (uaid and channel_id and message_id)` but not a show stopper on late fri.